src/mpi/init/netloc_util.c

```diff
@@ -10,6 +10,20 @@
 #ifdef HAVE_NETLOC
 #include "netloc_util.h"
 #include "mpl.h"
+#include <math.h>
```

Comment:
What are we relying on `math.h` for?  Is there a dependency on `libm`?

Reply:
I am making a call to ceilf in line 1406.

Reply:
Can you also answer my second question?

Reply:
The c++ examples have an explicit dependence on math. Otherwise, we don't seem to be linking against libm. Sorry I missed looking into this. I'll fix it.